src/app/tide/peptide_mods3.cc

```diff
@@ -396,15 +378,17 @@ class ModsOutputterAlt : public IModsOutputter {
     }
   }
 
-  string tempDir_;
   const vector<const pb::Protein*>& proteins_;
-  VariableModTable* modTable_;
-  map<int, int> modMaxCounts_;
-  int maxMods_;
-  HeadedRecordWriter* writer_;
-
-  map< int, pair<string, RecordWriter*> > tempFiles_; // id -> file, writer (ids must be in ascending order of mass)
-  uint64_t totalWritten_;
+  VariableModTable* mod_table_;
```

Comment:
why are there variables declared at the beginning and the end of the private class section of ModsOutputter? Doesn't it make more sense for all variables to be together at the beginning of the private section?

Reply:
I moved them to the beginning.